export/aovs.py

```diff
@@ -132,6 +132,9 @@ def convert(exporter, scene, context=None, engine=None):
         props = utils.create_props(prefix, definitions)
         props.Set(pipeline_props)
 
+        if (use_adaptive_sampling and final):
+            props.Set(pyluxcore.Property("film.noiseestimation.index", adaptive_sampling_film_pipeline_index))
+
```

Comment:
I would undo the changes in this file, my fixes were enough in my opinion.
I moved this code to line 130.